### ISSUE TITLE
Lots of keys makes things very slow

### ISSUE THREAD

**Issue by eyeseast** [eyeseast/dorchester#32]
Running this now:

```sh
dorchester plot data/ma-2010-race.geojson data/ma-2010-race-points.csv --progress \
	  -k White \
	  -k "Black or African American" \
	  -k "American Indian and Alaska Native" \
	  -k Asian \
	  -k "Native Hawaiian and Other Pacific Islander" \
	  -k Other \
	  -k "Two or More Races"
```

And it's basically running the whole process seven times, once for each key. That's no good. Probably going to mean clustering because lots of populations will round to zero. Oops. Better rethink some of this.

**Comment by eyeseast** [comment]
I wonder if I could sum the population from all keys, generate a list of points for a feature, and then match that to keys proportionally.

**Comment by eyeseast** [comment]
First pass at fixing this is breaking the tests that expect an error report for each feature/group combination. But if I'm fixing errors, maybe that doesn't matter anymore.

**Comment by eyeseast** [comment]
Fixed via #34
